ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.h

```diff
@@ -9,8 +9,9 @@
 @import Foundation;
 #import "React/RCTBridgeModule.h"
 
+@protocol RCTBridgeModule;
 RCT_EXTERN const NSNotificationName _Nonnull RNKiwiAccountDeletedNotification;
 
-@interface RNKiwiAccountUpdater : NSObject <RCTBridgeModule>
+@interface RNAccountManager : NSObject <RCTBridgeModule>
```

Comment:
I specifically named this class so that it wouldn't be a "manager". Why would you change this?

Reply:
So that we have the same name for android and ios